src/game/common/system/gametype.h

```diff
@@ -198,6 +198,17 @@ enum CellShroudStatus
 enum TerrainDecalType
 {
     TERRAIN_DECAL_0,
+    TERRAIN_DECAL_1,
+    TERRAIN_DECAL_2,
+    TERRAIN_DECAL_3,
+    TERRAIN_DECAL_4,
+    TERRAIN_DECAL_5,
+    TERRAIN_DECAL_6,
+    TERRAIN_DECAL_7,
+    TERRAIN_DECAL_8,
+    TERRAIN_DECAL_9,
+    TERRAIN_DECAL_A,
+    TERRAIN_DECAL_B,
```

Comment:
I haven't found any better names.

Reply:
so that corresponds to the TerrainDecalTextureName array, array tells us 0, TERRAIN_DECAL_DEMORALIZED_OBSOLETE
others for now comment with the name in array, 8 and 9 comment as related to fake kinds, A seems TERRAIN_DECAL_COUNT unless you got proof of A and B

Reply:
Fixed.